1

```diff
@@ -0,0 +1,69 @@
+[15:52:31] Using gulpfile ~/Development/projects/amphtml/gulpfile.js
```

Comment:
Remove this file.

Reply:
woops... was wondering where my junk log file went.